### ISSUE TITLE
Investigate weird values of -10 in some variables

### ISSUE THREAD

**Issue by ld-archer** [Leeds-MRG/Minos#92]
Just solved one problem with it throwing up values of -10 for ncigs (not a missing value from raw data). Couldn't find the problem in the code but managed to solve it by adding -10 to the list of missing codes to be replaced by NA (function in `utils.r`).

Two problems noticed whilst refactoring the run scripts.

#### ncigs

During the estimation of the ncigs zero inflated poisson model in data prep, from the 2015-2016 model onwards it was failing due to some negative values in dependent variable (ncigs == -10). I couldn't find the source of the problem because it seems to be a data issue. I added a short term fix by adding -10 to the list of missing codes to be replaced by NAs, but we should investigate the input data a bit more and figure out why we are seeing these values (-10 is not a missing code used by Understanding Society). Note these values are present BEFORE any rounding or setting null values to 0, so this is not the cause.

#### JobSec

Modules including Loneliness and Tobacco are failing due to negative values in job_sec (-10). I suspected these were caused by the Labour module, but deactivating that component did not fix the problem. Instead for a short term fix I have removed job_sec from the prediction of Loneliness, Tobacco, Alcohol, Neighbourhood, MWB, and Labour. Need to remember to replace these when the underlying problem is fixed.

**Comment by RobertClay** [comment]
I've been putting -10 as a new missing code for items that are intentionally missing. job_sec as -10 implies they are not employed and were given job_sec -8. Its not actually missing and doing this improves locf imputation a bit. 

not sure why this is happening for ncigs. suggets bad estimation of ncigs in zip model or modification of error codes. best guess is lines 62-65 or tobacco_zip.R

**Comment by RobertClay** [comment]
Yeah removing it from locf forward fill seems to solve the problem. I need to go through the pipeline again really updating for a bunch of new variables.
